microsoft-edge/progressive-web-apps-chromium/whats-new/pwa.md

```diff
@@ -12,12 +12,59 @@ ms.date: 02/23/2022
 This page lists the updates to Progressive Web Apps (PWAs) and Web Apps from the Microsoft Edge Web Apps team.  To try new features, read these announcements.  To stay up to date with the latest and greatest features, download the [Microsoft Edge preview channels](https://www.microsoftedgeinsider.com/download) (Beta, Dev, and Canary).
 
 
+<!-- ====================================================================== -->
+## What's New in Microsoft Edge 105
+
+Microsoft Edge 105 brings the following new features for Progressive Web Apps.
+
+
+### Window Controls Overlay
+
+The **Window Controls Overlay** API is now available by default with Microsoft Edge 105. This feature makes it possible for web developers to take control of the area that would normally be used by the title bar of an installed PWA window. The PWA code can use this area to render arbitrary web content such as a custom title, navigation buttons, or a menu bar.
+
+This feature makes PWAs installed on desktop operating systems more similar to their native counterparts.
+
+The Window Controls Overlay API is available on Windows, macOS, and Linux operating systems.
+
+![A PWA, with a custom titlebar](media/custom-title-bar.png)
+
+To learn more, check out these resources:
+
+* [Display content in the title bar](../how-to/window-controls-overlay.md).
+* [Closing a 30 pixel gap between native and web](https://blogs.windows.com/msedgedev/2022/09/27/closing-pixel-gap-native-web-window-controls-overlay/).
+* [Window Controls Overlay API](https://developer.mozilla.org/docs/Web/API/Window_Controls_Overlay_API) on MDN.
+
+
+### Notifications from a PWA are now emitted from this PWA
+
+In earlier versions, the notifications created by PWAs looked like they were emitted by Microsoft Edge.
```

Comment:
```suggestion
In earlier versions, the notifications that were created by PWAs looked like they were emitted by Microsoft Edge.
```
to avoid mis-parsing as: The notifications created something.